train.py

```diff
@@ -84,14 +84,14 @@ def train_net(net,
 1. When do BasicDataset function(in src.unet_.dataset.py), image shpae become 5 dimension (2, 19, 681, 1086, 12) (batch size, yr, y, x, band).
```

Comment:
I guess we probably want to get rid of the year dimension 😉 but haven't looked at the `dataset.py` file in detail yet

Reply:
Got it.
And the processed data is based on @sdat2 's works on preprocessing data for xgboots. It will be clearer about the data if also have a look on unet.py in src.models and src.preprocessing folder.